### PR TITLE
refactor header symbols. seems to break the the macro system.

### PR DESCRIPTION
**Pull request by rhl-**
This doesn't build, not sure how to fix the macro system..

### CODE REVIEW

File: include/El/blas_like/level3.hpp [hunk 1]
```diff
@@ -9,621 +9,25 @@
...
-//       and is obviously not a BLAS routine. However, it is a basic linear
-//       algebra routine making use of Elemental's core data structures, and
-//       so this is the natural placement
-template<typename T>
-void Multiply
-( Orientation orientation,
-  T alpha, const SparseMatrix<T>& A, const Matrix<T>& X,
-  T beta,                                  Matrix<T>& Y );
-
-template<typename T>
```
Comment (poulson) [inline include/El/blas_like/level3.hpp]: It appears you have not replaced this prototype anywhere and that this is the problem.

Comment (rhl-) [inline include/El/blas_like/level3.hpp]: fixed, and the other two missing things at the bottom were added.

Comment (rhl-) [inline include/El/blas_like/level3.hpp]: things build locally.

**Comment by poulson** [comment]
According to the failed Travis build log, https://s3.amazonaws.com/archive.travis-ci.org/jobs/105338873/log.txt, the first error is that `AxpyContract` is not defined within `Symv.hpp`

**Pull request closed without merging**